main.py

```
@@ -0,0 +1,140 @@
+
+    Args:
+        input_path: путь к входному видео файлу
+
+    Returns:
+        путь к выходному файлу (processed_<имя_входного_файла>.mp4)
+    """
+    # Получаем директорию и имя входного файла
+    input_dir = os.path.dirname(input_path) or '.'
+    input_filename = os.path.basename(input_path)
+
+    # Формируем путь к выходному файлу с префиксом "processed_"
```

Ruff (0.14.4)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)